### ISSUE TITLE
Support for Scikit-Survival models that is compatible with Sklearn?

### ISSUE THREAD

**Issue by WeijiaZhang24** [jpmml/sklearn2pmml#332]
Is is possible to export models trained using Scikit-Survival (sksurv)? 
This is the repos for sksurv:  https://github.com/sebp/scikit-survival

sksurv contains a RandomSurvivalForest algorithm which extend RandomForest to right-censored survival data. 
In standard RandomForest, the regression target y is a number, but in survival data , the labels are in the form of [time, event_indicator]. If event_indicator==1, then time is the same as y (event is observed); however, when event_indicator == 0, we only know taht y>time (event is not observed up to the observed time).

Any help would be appreciated!

**Comment by vruusmann** [comment]
> Is is possible to export models trained using Scikit-Survival (sksurv)?

I'm going to explore your earlier [XGBoost example](https://github.com/jpmml/jpmml-xgboost/issues/63) a bit in order to gain a better understanding about the state-of-the-art in survival analysis.

The fundamental problem here is that "survival" appears to be a different endpoint than "regression".

The PMML specification does not provide a dedicated "survival" mining function type: https://dmg.org/pmml/v4-4-1/GeneralStructure.html#xsdType_MINING-FUNCTION

The obvious fix would be to define a new mining function type ourselves. I guess it's safe to say today that it's not reasonable to count on Data Mining Group's help here, because they're largely non-operational (still waiting to receive an initial feedback on some feature requests that I posted to them 1+ year ago).

> sksurv contains a RandomSurvivalForest algorithm which extend RandomForest to right-censored survival data.

The JPMML-SkLearn library already provides a PMML converter for the RandomForest class.

RandomSurvivalForest and RandomForest should use identical tree ensemble data structures. Therefore, it would be build a PMML converter for RandomSurvivalForest by simply applying some post-processing to RandomForest prediction.
